### PR TITLE
docker-confluence-for-testing-4 Provide ability to customize locale a…

### PR DESCRIPTION
**Pull request by aruizca**
Provide ability to customize locale and timezone on the running container

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -22,13 +27,13 @@ services:
     image: postgres:9.4
     container_name: postgres
     hostname: postgres
+    environment:
```
Comment (aruizca) [inline docker-compose.yml:30]: Just moved

File: scripts/run-confluece-container.sh [hunk 2]
```diff
@@ -1,20 +1,19 @@
-#!/bin/bash
+#!/usr/bin/env bash
 #-e  Exit immediately if a command exits with a non-zero status.
 set -e
 
 function usage
 {
     local scriptName=$(basename "$0")
-    echo "usage: ${scriptName} ENV=VALUE ENV2=VALUE"
+    echo "usage: ${scriptName} x.y.z ENV=VALUE ENV2=VALUE"
```
Comment (aruizca) [inline scripts/run-confluece-container.sh:8]: No need for the user to know that the main environment variable is called CONFLUENCE_VERSION

File: scripts/run-confluece-container.sh [hunk 3]
```diff
@@ -33,10 +35,16 @@ function parse_args
     
     # restore positional args
     set -- "${args[@]}"
+    shift 1
 }
 # Set current folder to parent
 cd "$(dirname "$0")"/..
 
+if [[ ! -z "$CONFLUENCE_VERSION" ]]
```
Comment (oskar-flores) [inline scripts/run-confluece-container.sh:43]: While not strictly necessary, is good practice to surround variable with curly braces.
This way variable expand is less error prone, and also is an easy way to see the difference between bash variable and env variable

File: docker-compose.yml [hunk 4]
```diff
@@ -8,6 +8,11 @@ services:
       context: .
       args:
           CONFLUENCE_VERSION: ${CONFLUENCE_VERSION}
+    environment:
+      - TZ
```
Comment (oskar-flores) [inline docker-compose.yml:12]: Is there any default in here?
I think is good idea to add defaults to the .env file.
IMHO,  that gives two good things:
- Is documented
- Is asy to change even you dont really know how to work with docker environents

**Review by oskar-flores — APPROVED** [review]
Good job,
I think passing variable to the container, is in th eend a good idea.
Just added minor comments, as a suggestion, but PR is approved

File: docker-compose.yml (hunk 4 above)
Comment (aruizca) [reply docker-compose.yml:12]: I agree. The default should be the host timezone and locale. Any idea how to resolve that dynamically

Comment (aruizca) [reply docker-compose.yml:12]: The good thing is that those variables are only passed if there is an environment variable in the host with the same name. So I am going to assume that if there is one available by default is the host one

**Pull request merged**